src/services/shims.ts

```diff
@@ -467,7 +467,7 @@ namespace ts {
         }
     }
 
-    export class CoreServicesShimHostAdapter implements ParseConfigHost, ModuleResolutionHost {
+    export class CoreServicesShimHostAdapter implements ParseConfigHost, ModuleResolutionHost, JsTyping.TypingResolutionHost {

```

Comment:
Another seemingly unrelated change in this PR?

Is this a driveby bugfix, or actually related to the `depth` issue?